<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1122</width>
    <height>863</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="preferences">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>10</y>
     <width>101</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Set Preferences</string>
   </property>
  </widget>
  <widget class="QPushButton" name="email">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>10</y>
     <width>91</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Email Alerts</string>
   </property>
  </widget>
  <widget class="QPushButton" name="change">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>10</y>
     <width>111</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Change Password</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="grp1">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>270</y>
     <width>551</width>
     <height>291</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="txt1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>511</width>
      <height>211</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="ic1">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="grp4">
   <property name="geometry">
    <rect>
     <x>549</x>
     <y>559</y>
     <width>571</width>
     <height>291</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="txt4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>521</width>
      <height>191</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="ic4">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="grp3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>560</y>
     <width>551</width>
     <height>291</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="txt3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>511</width>
      <height>211</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="ic3">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="grp2">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>270</y>
     <width>571</width>
     <height>291</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="txt2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>511</width>
      <height>211</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="ic2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="next">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </widget>
  <widget class="QLabel" name="micon">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>1091</width>
     <height>221</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="movie">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter a movie</string>
   </property>
  </widget>
  <widget class="QPushButton" name="imdb">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>10</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>imdb search</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="cpass">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>10</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>New Password</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="D:/d/test.qrc"/>
 </resources>
 <connections/>
</ui>
